ee/docker-compose.ch.arm64.yml

```diff
@@ -0,0 +1,98 @@
+version: '3'
+
+services:
+    db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
+        ports:
+            - '5432:5432'
+    redis:
+        image: 'redis:alpine'
+        ports:
+            - '6379:6379'
+    clickhouse:
+        # Build with: yarn arm64:build:clickhouse
+        image: clickhouse-dev-arm64:latest
```

Comment:
Is there a simple way we can avoid having another docker compose file? Ie choosing an image based on arch env var, or using a multi arch image?

Reply:
> choosing an image based on arch env
I looked, but didn't find any way to do that. I think that's intentional as well.

Reply:
You could do something like `image: "${CLICKHOUSE_IMAGE:-yandex/clickhouse-server:21.6.5}"` so we'd default to yandex if `CLICKHOUSE_IMAGE` isn't set. Then just set this env var in the package.json scripts for arm.